client/post-editor/editor-mobile-navigation/index.jsx

```diff
@@ -2,18 +2,19 @@
  * External dependencies
  */
 const React = require( 'react' );
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-const Site = require( 'blocks/site' ),
-	Gridicon = require( 'components/gridicon' ),
-	layoutFocus = require( 'lib/layout-focus' );
+const Site = require( 'blocks/site' );
```

Comment:
Agreed. I was just trying to make as few changes as possible, but I guess I might as well here since they are already es6.